src/components/cc-orga-member-card/cc-orga-member-card.js

```diff
@@ -0,0 +1,531 @@
+const profileSvg = new URL('../../assets/profile.svg', import.meta.url).href;
+
+const BREAKPOINT_MEDIUM = 740;
+const BREAKPOINT_SMALL = 580;
+const BREAKPOINT_TINY = 350;
+
+/**
+ * @typedef {import('./cc-orga-member-card.types.js').Authorisations} Authorisations
+ * @typedef {import('./cc-orga-member-card.types.js').OrgaMemberCardState} OrgaMemberCardState
+ */
+
+/**
+ * A component showing information about a member from a given organisation.
+ *
+ * With the right authorisations:
+ * - This component provides a way to delete the member from the organisation.
+ * - This component provides a way to edit the role of the member within a given organisation.
+ *
+ * ## Technical Details
+ *
+ * This component heavily relies on `cc-stretch` to make sure all cards look the same whatever the role and MFA status may be.
+ * This component also heavily relies on CSS `grid` and the `withResizeObserver` mixin to switch from a "table" like design to a card design when the card width shrinks.
+ *
+ * @cssdisplay block
+ *
+ * @event {CustomEvent<ToggleEditing>} cc-orga-member-card:toggle-editing - Fires the `id` of the member related to the card and the new state (`editing` or `loaded`) to specify when card is in edit mode or not.
+ * This allows the list component to close all other cards in edit mode to leave only one in edit mode at once.
+ * @event {CustomEvent<DeleteMember>} cc-orga-member-card:delete - Fires the `id` of the member to be removed from the org with their `name` or `email`.
+ * @event {CustomEvent<DeleteMember>} cc-orga-member-card:leave - Fires the `id` of the member to be removed from the org with their `name` or `email`.
+ * We don't fire a delete event so that it can be processed differently by the smart component (leaving the org means the user has to be redirected).
+ * @event {CustomEvent<UpdateMember>} cc-orga-member-card:update - Fires the `id` of the member, their new `role` and their `name` or `email`.
+ */
+export class CcOrgaMemberCard extends withResizeObserver(LitElement) {
+
+  static get properties () {
+    return {
+      authorisations: { type: Object },
+      member: { type: Object },
+      _size: { type: String, state: true },
+      _newRole: { type: String, state: true },
+    };
+  }
+
+  constructor () {
+    super();
+
+    /** @type {Authorisations} Sets the authorisations that control the display of the edit / delete buttons. */
+    this.authorisations = {
+      edit: false,
+      delete: false,
+    };
+
+    /** @type {OrgaMemberCardState} Sets the state and data of the member. */
+    this.member = {
+      state: 'loaded',
+      id: '',
+      email: '',
+      role: '',
+      isMfaEnabled: false,
+      isCurrentUser: false,
+    };
+
+    /** @protected */
+    this.breakpoints = {
+      // used to switch to buttons wrapping below when component width < 740 and vertical card layout when < 580, buttons and badges below each other when < 350.
```

Comment:
This line is quite hard to read could we replace `<` and `>` by below and above and make the sentence more readable?
I would suggest something like this but its up to you:
```suggestion
     /* used to: 
     * * wrap buttons when the component width is below 740.
     * * have a vertical card layout when the width is below 580.
     * * have buttons and badges below each other when the width is below 350.
     */
```